src/ipc.cpp

```diff
@@ -188,7 +189,6 @@ bool IPC::waitUntilAccepted(time_t postTime, int32_t timeout/*=-1*/)
     while (!(result = isEventAccepted(postTime)))
     {
         qApp->processEvents();
-        QThread::msleep(10);
```

Comment:
No, `qApp->processEvents();` prevents it. Rather than that, qTox starts immediately the second you open it… :)